src/vendor/registry_shim/registry_shim.go

```diff
@@ -0,0 +1,23 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package registry_shim implements the ProvisioningAppliance:RegisterDevice RPC.
+package registry_shim
+
+import (
+	"context"
+	"log"
+
+	pap "github.com/lowRISC/opentitan-provisioning/src/pa/proto/pa_go_pb"
+	diu "github.com/lowRISC/opentitan-provisioning/src/proto/device_id_utils"
+	pb "github.com/lowRISC/opentitan-provisioning/src/proxy_buffer/services/proxybuffer"
+)
+
+func RegisterDevice(ctx context.Context, buffer pb.Registry, request *pap.RegistrationRequest) (*pap.RegistrationResponse, error) {
+	log.Printf("In PA - Received RegisterDevice request with DeviceID: %v", diu.DeviceIdToHexString(request.DeviceData.DeviceId))
```

Comment:
Should we add a non-implemented error here?

Reply:
done